common/lib/dependabot/metadata_finders/base/changelog_finder.rb

```diff
@@ -271,6 +271,7 @@ def fetch_bitbucket_file_list
         end
 
         def fetch_gitlab_file_list
+          branch = default_gitlab_branch
```

Comment:
Not really important, but since `default_gitlab_branch` is already memoized we don't really need to assign it here and could use `default_gitlab_branch` directly below? 🤷

Reply:
You are absolutely right. The reason I did it like this is so that it is the same as in the [fetch_bitbucket_file_list](https://github.com/dependabot/dependabot-core/blob/main/common/lib/dependabot/metadata_finders/base/changelog_finder.rb#L252)-method. I would be happy to change this as suggested, please let me know 😄

Reply:
It's fine :D